CONTRIBUTING.md

```diff
@@ -136,140 +155,169 @@ The process described here has several goals:
 
 Please follow these steps to have your contribution considered by the maintainers:
 
-1. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
+1. Follow all instructions in [Pull Request template](./Pull_Request_Template.md)
 2. Follow the [coding guidelines](#coding-guidelines)
-3. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
+3. After submitting your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing, for example:
+
+![Status OK](/.github/assets/git_status_check_ok.png "Status OK")
```

Comment:
Would it be possible to have a bigger screenshot so that there is a bit more context?

Reply:
Also, these screenshot were taking using darkmode, is there a way to provide non-darkmode images as well?